scripts/add_electricity.py

```diff
@@ -177,50 +175,54 @@ def costs_for_storage(store, link1, link2=None, max_hours=1.0):
 
 
 def attach_wind_and_solar(
-    n, costs, input_profiles, tech_modelling, extendable_carriers
+    n, costs, number_microgrids, input_profiles, tech_modelling, extendable_carriers
```

Comment:
Rather than the list of number of microgrids, we may pass here the bus_names where wind and solar shall be attached, or we need an automatic flag that enables understanding that.
This would avoid hard-coding names inside this function that leads to duplication.

Reply:
For example, the name of the bus could be the name of the microgrid itself.